docs/3.Custom Policies/YAML Custom Policies.md

```diff
@@ -45,6 +46,70 @@ metadata:
   guideline: "https://docs.bridgecrew.io/docs/ckv2_custom_1"
 ```
 
+### Extensions
+
+Extensions offer a way to enrich the data of the original resource to support use cases, which are not possible with given data.
+
+Available extensions:
+
+* IAM_ACTION_EXPANSION (enabled by default)
```

Comment:
WDYT about making it the inverse logic for this flag? IAM Action Expansion is always on unless you use extension: DISABLE_IAM_ACTION_EXPANSION?